src/System.CommandLine/Argument.cs

```diff
@@ -108,7 +108,7 @@ public List<ISuggestionSource> Suggestions
         }
 
         /// <summary>
-        /// Gets or sets the <see cref="System.Type" /> that the argument will be converted to.
+        /// Gets or sets the <see cref="Type" /> that the argument will be converted to.
```

Comment:
```suggestion
        /// Gets or sets the <see cref="Type" /> that the argument value will be converted to.
```

Reply:
Instead of "value" how about "token(s)"?

I feel like "value" is also ambiguous, per your comment above. Does it refer to a value on the command line or a typed, converted value after parsing is done?

Reply:
Yea token would be a better word. I keep looking at the terminology docs you wrote up and comparing with those to make sure things are consistent.